python/lsst/analysis/tools/actions/plot/wholeSkyPlot.py

```diff
@@ -58,8 +69,6 @@ class WholeSkyPlot(PlotAction):
     xLimits = ListField[float](doc="Plotting limits for the x axis.", default=[-5.0, 365.0])
     yLimits = ListField[float](doc="Plotting limits for the y axis.", default=[-10.0, 60.0])
     autoAxesLimits = Field[bool](doc="Find axes limits automatically.", default=True)
-    dpi = Field[int](doc="DPI size of the figure.", default=500)
-    figureSize = ListField[float](doc="Size of the figure.", default=[9.0, 3.5])
```

Comment:
I have removed these because the figures are getting very large and taking a long time to get from the butler. The figure size now also needs to be very specific and if you change it nothing will line up. If this becomes a problem for @erinleighh then I will make a better fix on another ticket in the future.